src/test/functional/serialization.js

```diff
@@ -0,0 +1,15 @@
+import { createTestORM } from "../helpers";
+
+let orm;
+
+beforeEach(() => {
+    orm = createTestORM();
+});
+
+it("can serialize and deserialized the state", () => {
```

Comment:
This test fails before the change.
Let me know if there's a better place to put the test.